### PR TITLE
Update documentdb-performance-levels.md

### PR DESCRIPTION
**Pull request by johnfmacintyre**
fix typo

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@johnfmacintyre__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (johnmac). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: articles/documentdb/documentdb-performance-levels.md [hunk 1]
```diff
@@ -13,7 +13,7 @@
 	ms.tgt_pltfrm="na" 
 	ms.devlang="na" 
 	ms.topic="article" 
-	ms.date="01/29/2016" 
+	ms.date="03/6/2016"
```
Comment (mimig1) [inline articles/documentdb/documentdb-performance-levels.md]: The day has to be two digits, please change to 03/06/2016.

**Comment by mimig1** [comment]
@mimig1 - tagging myself to get updates on this PR

**Comment by johnfmacintyre** [comment]
fixed date

**Comment by mimig1** [comment]
#sign-off

**Pull request merged**